pkg/security/probe/probe_bpf.go

```diff
@@ -179,7 +181,11 @@ func (p *Probe) InitManager(rs *rules.RuleSet) error {
 
 // Start the runtime security probe
 func (p *Probe) Start() error {
-	return p.manager.Start()
+	if err := p.manager.Start(); err != nil {
+		return err
+	}
+	go p.loadController.Start(context.Background())
```

Comment:
Any reason to pass the `Context` if not used by `Start`?

Reply:
I might be missing something but I'm creating a new context in `Start` that uses the provided context as parent.

Reply:
Right, I meant you could initialize it inside the `Start`, looks like the caller is not interested in controlling with cancel or anything like that.